internal/characters/keqing/skill.go

```diff
@@ -7,7 +7,9 @@ import (
 	"github.com/genshinsim/gcsim/pkg/core/combat"
 )
 
-var skillFrames []int
+var (
+	skillFrames []int
```

Comment:
NIT: this should just be single line now (which means reverting to the original)